snuba/util.py

```diff
@@ -69,6 +70,70 @@ def time_expr(alias, granularity):
 
     return template.format(column=column, granularity=granularity)
 
+def function_expr(fn, args_expr=''):
+    """
+    Generate an expression for a given function name and an already-evaluated
+    args expression. This is a place to define convenience functions that evaluate
+    to more complex expressions.
+
+    """
+    # For functions with no args, (or static args) we allow them to already
+    # include them as part of the function name, eg, "count()" or "sleep(1)"
+    if not args_expr and fn.endswith(')'):
+        return fn
```

Comment:
@fpacifici that's true only if args are not passed in. I'll add an extra test. 
```
"aggregations": [
    ["uniq", "tags[environment]", "unique_envs"],
],
```
In this case the relevant call will be: 
`column_expr('tags[environment]', {}, alias='unique_envs', aggregate='uniq')`
which in turn calls `function_expr(fn='uniq', args_expr='environment')`  because it checks that `args_expr` doesn't have anything, it'll bypass that step.